src/libraries/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -3,8 +3,9 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Is System.Private.CoreLib already generating a documentation file? We need it to generate it and disable warnings until we port all the documentation to System.Private.CoreLib.

Reply:
I wanted to confirm this with you.

S.P.CoreLib does generate docs, but they are not yet made mandatory. I assume this change needs to be reverted, and the `GenerateDocumentationFile` change needs to be the last change we add to CoreLib, after all APIs that live there have their documentation backported.

Reply:
I think having the `GenerateDocumentationFile=true` here is valuable as a future proof in the case where we add public API to this assembly and the implementation were to live here as opposed to System.Private.CoreLib.

We will have to add some infrastructure to merge the System.Private.CoreLib.xml file into the reference assemblies that expose those APIs xmls.

Reply:
Okay thanks. If you think it's safe to keep it, then I'll keep it.